Spec/RealtimeClientConnection.swift

```diff
@@ -3028,26 +2989,165 @@ class RealtimeClientConnection: QuickSpec {
                     default:
                         fail("unhandled: \(fixtureMessage["expectedType"].string!)")
                     }
+                }
 
-                    waitUntil(timeout: testTimeout) { done in
-                        channel.publish([message]) { err in
-                            if let err = err {
-                                fail("\(err)")
+                // https://github.com/ably/wiki/issues/22
+                it("should encode and decode fixture messages as expected") {
+                    let options = AblyTests.commonAppSetup()
```

Comment:
Should this not explicitly use `JSON` i.e. `binaryProtocol = false` and should the test description be updated to reflect that these tests are about JSON based interoperability?